test/images/scripts/lib/make-srpm

```diff
@@ -4,28 +4,21 @@ set -eu
 
 tar=$1
 
-version=$(echo $1 | sed -n 's|.*cockpit-\([^ -/]\+\)\.tar\..*|\1|p')
+version=$(echo "$1" | sed -n 's|.*cockpit-\([^ -/]\+\)\.tar\..*|\1|p')
 if [ -z "$version" ]; then
     echo "make-srpm: couldn't parse version from tarball: $1"
     exit 2
 fi
 
+# We actually modify the spec to the srpm is standalone buildable
```

Comment:
Typo? Not sure what this is supposed to be... If you write what this is supposed to be, I can change while merging.